Misc/NEWS.d/next/Library/2022-05-08-19-21-14.gh-issue-84131.rG5kI7.rst

```diff
@@ -0,0 +1,3 @@
+The :class:`pathlib.Path` deprecated method ``link_to`` has been removed.
+Use 3.10's :meth:`pathlib.Path.hardlink_to` method instead as its semantics
+are consistent with that of :meth:`pathlib.Path.symlink_to`.
```

Comment:
```suggestion
Use 3.10's :meth:`~pathlib.Path.hardlink_to` method instead as its semantics
are consistent with that of :meth:`~pathlib.Path.symlink_to`.
```

Adds tilde (`~`) characters to suppress the `pathlib.Path.` prefix in the link text, as we've already introduced the `pathlib.Path` namespace in the first line.